fission/environment.go

```diff
@@ -319,6 +323,20 @@ func getResourceReq(c *cli.Context, resources v1.ResourceRequirements) v1.Resour
 		limitResources[v1.ResourceMemory] = memLimit
 	}
 
+	limitCPU := limitResources[v1.ResourceCPU]
+	requestCPU := requestResources[v1.ResourceCPU]
+
+	if limitCPU.Cmp(requestCPU) < 0 {
+		log.Fatal(fmt.Sprintf("MinCPU (%v) cannot be greate than MaxCPU (%v)", limitResources[v1.ResourceCPU], requestResources[v1.ResourceCPU]))
```

Comment:
The problem here is if that the user specifies a request but not a limit, this errors out.  How about just setting max=min if max is zero.